metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -352,21 +359,37 @@ def _get_dataset_column_mean(
     def _get_dataset_column_median(
         self, column_profile: DatasetFieldProfileClass, column: str
     ) -> None:
-        if self.config.include_field_median_value:
+        if not self.config.include_field_median_value:
+            return
+        try:
             column_profile.median = str(self.dataset.get_column_median(column))
+        except Exception as e:
+            logger.debug(f"Exception was {e}")
```

Comment:
"Caught exception while attempting to get column median for column {column}. {e}